### ISSUE TITLE
rent_exempt Anchor Constraint

### ISSUE THREAD

**Issue by 0xIchigo** [coral-xyz/anchor#2783]
The [Anchor account constraints documentation](https://docs.rs/anchor-lang/latest/anchor_lang/derive.Accounts.html) outlines the `rent_exempt` account constraint. This constraint enforces rent exemption when set to `enforce` and skips the rent exemption check normally done through other constraints with `skip`

With recent updates, [Solana no longer allows the creation of non-rent-exempt accounts](https://docs.solana.com/developing/intro/rent#rent-exempt). This makes the `rent_exempt` constraint obsolete

**Comment by acheroncrypto** [comment]
Yes, it will be removed in https://github.com/coral-xyz/anchor/pull/2708

**Comment by 0xIchigo** [comment]
Ah sounds good, thanks! @acheroncrypto
